### PR TITLE
Taofeeq/futr 356 add rich text input field for email body in workflow api

### PR DESCRIPTION
**Pull request by taofeeq-deru**
Screenshot 2024-09-10 at 8 47 50 PM

### CODE REVIEW

File: packages/admin/src/components/editor/extensions/extensions.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    HTMLAttributes: {
+      class: 'leading-normal -mb-2',
+    },
+  },
+  blockquote: {
+    HTMLAttributes: {
+      class: 'border-l-4 border-stone-700',
+    },
+  },
+  codeBlock: {
```
Comment (adeleke5140) [inline packages/admin/src/components/editor/extensions/extensions.ts]: we don't have codeBlock support in the mail so I believe we can take this out, also horizontalRule, taskList, task item

**Review by adeleke5140 — COMMENTED** [review]
good one, left some comments

File: packages/admin/src/components/editor/extensions/extensions.ts (hunk 1 above)
Comment (taofeeq-deru) [reply packages/admin/src/components/editor/extensions/extensions.ts]: okay, on it

**Review by adeleke5140 — APPROVED** [review]
lgtm

**Pull request merged**